src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -99,7 +109,7 @@ private Loan(JsonObject representation, Item item, User user, User proxy,
   public static Loan from(JsonObject representation) {
     defaultStatusAndAction(representation);
     return new Loan(representation, null, null, null, null, null, null,
-      LoanPolicy.unknown(null), null);
+      null, null);
```

Comment:
I don't think that policies should ever be null, unless we intend to check this everywhere it is used?

Reply:
there are default initialization for policies in the constructor of the loan

Reply:
Hmm, ok, maybe I'm misunderstanding this.

If the policies instance is initialised within the constructor, what is this null parameter for?

Reply:
Fixed